src/commcare_cloud/ansible/deploy_stack.yml

```diff
@@ -2,6 +2,7 @@
 # deploy-stack.yml
 
 - import_playbook: update_apt_cache.yml
+- import_playbook: bootstrap_python.yml
```

Comment:
Am I understanding correctly that this will put Python 3.10 on every machine, not just the control machine? Is that necessary? I think we only need it on the control machine where commcare-cloud/ansible runs.

Reply:
Yeah that's correct, and also correct that it is not necessary to install on every machine, only machines where commcare-cloud runs. I believe ansible does execute on all machines (which is why the raw ansible module is significant), but ansible makes an effort to maintain compatibility with a wide range of python versions so it isn't necessary to target a specific version.

With that in mind, I'm pretty sure only the control machine runs commcare-cloud, and therefore we could limit this playbook to control hosts only? I'm operating with the understanding that "control" is a standard across commcare-cloud env setups.

Reply:
Confirmed with Danny that I can assume all envs will have a "control" group, so I think bb604efd67566cf6d0abe8a02cb8b9a9977e12a8 effectively limits this playbook to running on servers within that group only.

Reply:
Ok finally got this into a working state thanks to @skodde! Marco brought up a good point as well related to this thread, which is that even though commcare-cloud does not run on the non-control machines, and ansible supports a large range of python versions, it could still be good to upgrade these non-control machines to 3.10 now and see if our regular running of ansible roles are impacted at all. This would be one less concern when upgrading to ubuntu 22.04 which at that point would force these machines onto 3.10.